lib/filters/organization_filter.rb

```diff
@@ -0,0 +1,19 @@
+module ActiveInteraction
+  class OrganizationFilter < Filter
+    register :organization
+
+    def cast(value, context)
+      schema = OrganizationUpdateSchema.schema[:properties]
+      included_keys = value.keys.map(&:to_sym)
+      if included_keys.all? {|s| schema.key? s } && valid?(value)
```

Comment:
Does this work for nested hashes and arrays? My own thought had been to actually call the `OrganizationSchema.validate!(value)`, similarly to how `JsonSchemaValidator` does it (which provides the `params_for` that is used by the create and update actions.

I'm fine with your approach, but do want to check if it works in all cases. You could simply add tests for that on the `Organizations::Create` operation.